docs/release_notes/v1.11.5.md

```diff
@@ -1,6 +1,36 @@
 # Dapr 1.11.5
 
-This patch release includes a fix in gRPC-to-gRPC service invocation when the `ServiceInvocationStreaming` feature is enabled. Users who rely on gRPC-to-gRPC service invocation (legacy, non proxy-based) are encouraged to upgrade to Dapr 1.11.5 before upgrading existing deployments to 1.12.x, to prevent temporary issues during a rollout of Dapr 1.12.x.
+This update contains the following security fixes:
+
+- Security: prevent Sentry and Injector from applying the `daprsystem` Configuration from a non control plane namespace.
+
+Additionally, this patch release contains bug fixes:
+
+- Fixed: gRPC-to-gRPC service invocation when the `ServiceInvocationStreaming` feature is enabled. Users who rely on gRPC-to-gRPC service invocation (legacy, non proxy-based) are encouraged to upgrade to Dapr 1.11.5 before upgrading existing deployments to 1.12.x, to prevent temporary issues during a rollout of Dapr 1.12.x.
+- Fixed: Fix invoke remote stream from closing stream before EOF
+- Fixed: Downgrading from 1.12 to 1.11
+- Fixed: Return HTTP status code in HTTP service invocation with resiliency enabled
+
+## Security: Sentry and Injector only apply `daprsystem` Configuration from the control plane namespace
+
+### Problem
+
+Sentry and Injector will apply the `daprsystem` configuration from a non-control plane namespace if the namespace name is alphabetically higher than the control plane namespace name.
+
+### Impact
+
+Accidentally or maliciously, a Kubernetes user can write a Configuration in a non-control plane namespace that will be applied by Sentry and Injector.
+This can re-write the Sentry CA, disable mTLS, or otherwise bring down the entire Dapr cluster.
+
+### Root cause
+
+Sentry and Injector currently _list_ Configurations, before matching on the list for the `daprsystem` Configuration.
+
+### Solution
+
+Update Sentry and Injector to only _get_ the `daprsystem` Configuration, instead of listing all Configurations.
```

Comment:
```suggestion
Update Sentry and Injector to only get the `daprsystem` Configuration from the namespace where the Dapr control plane is installed, instead of listing all Configurations.
```